source/components/form/form.js

```diff
@@ -0,0 +1,79 @@
+const inputs = document.querySelectorAll('.cu-input');
```

Comment:
We should wrap this in an anonymous function to avoid polluting the global namespace.

http://markdalgleish.com/2011/03/self-executing-anonymous-functions/